src/analysis/normed_space/operator_norm.lean

```diff
@@ -0,0 +1,220 @@
+/-
+Copyright (c) 2019 Jan-David Salchow. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jan-David Salchow
+
+The space of bounded linear maps
+
+Define the set of bounded linear maps between normed spaces and show basic facts about it. In
+particular
+
+(*) define a set L(E,F) of bounded linear maps between k normed spaces,
+(*) show that L(E,F) is a vector subspace of E → F,
+(*) define the 'operator norm' on L(E,F) and show that it induces the structure of a normed space
+    on L(E,F).
+-/
+import algebra.module
+import analysis.normed_space.bounded_linear_maps
+
+variables {k : Type*} 
+variables {E : Type*} {F : Type*}
+
+
+-- Define the subspace of bounded linear maps, introduce the notation L(E,F) for the set of bounded linear maps.
+section bounded_linear_maps
+
+variables [hnfk : normed_field k] [normed_space k E] [normed_space k F]
+include hnfk
+
+def bounded_linear_maps : subspace k (E → F) :=
+{ carrier := {A : E → F | is_bounded_linear_map A},
+  zero := is_bounded_linear_map.zero,
+  add := assume A B, is_bounded_linear_map.add,
+  smul := assume c A, is_bounded_linear_map.smul c }
+
+notation `L(` E `,` F `)` := @bounded_linear_maps _ E F _ _ _
+
+/-- Coerce bounded linear maps to functions. -/
+instance bounded_linear_maps.to_fun : has_coe_to_fun $ L(E,F) :=
+{F :=  λ _, (E → F), coe := (λ f, f.val)}
+
+@[extensionality] theorem ext {A B : L(E,F)} (H : ∀ x, A x = B x) : A = B :=
+begin
```

Comment:
the `ext` tactic should do large parts of this.
There is already setup for the `subspace` and then it is talking about `set`s.

Reply:
set_coe.ext is my new best friend, the setup for submonoids doesn't seem to apply directly though